Cargo.toml

```diff
@@ -8,8 +8,10 @@ authors = ["Heath Stewart (https://github.com/heaths)"]
 publish = false
 
 [dev-dependencies]
+azure_core = "*"
 azure_identity = "*"
 azure_security_keyvault_secrets = "*"
+futures = "*"
```

Comment:
I recommend a fix dependency on 3P crate versions. I only use "*" for ours to help make sure these examples keep working as we release new versions. That's on us to fix. Let's not risk 3P breakage.